commandline/WSL/faq.md

```diff
@@ -122,6 +123,28 @@ Mount points for hard drives on the local machine are automatically created and
  
 Example usage would be `cd /mnt/c` to access c:\
 
+### How do I use a Windows file with a Linux app?
+
+One of the benefits of WSL is being able to use the same file with both Windows and Linux apps or tools.
+
+One of the main limitations of using WSL is that changing Linux files with a Windows app or tool is not allowed. See: [Do not change Linux files using Windows apps and tools](https://blogs.msdn.microsoft.com/commandline/2016/11/17/do-not-change-linux-files-using-windows-apps-and-tools/)
+
+However changing Windows files with a Linux app or tools is allowed.
+
+One way to use a Windows file with a Linux app is to use an absolute path, e.g. `/mnt/c/Users/<Windows User>/Documents/Projectes/<filename>`.
+
+However not all Linux apps or tools can access a file using `/mnt`.
+
+A solution is to create a [symbolic link](http://manpages.ubuntu.com/manpages/precise/man7/symlink.7.html).

```

Comment:
Could I suggest a little re-wording for those who may be unfamiliar with symlinks:

A solution is to create a symbolic link (...) to map a virtual Linux folder to a physical Windows folder. For example: To map a `/home/<Linux user>/Projects` virtual folder to the physical `C:\Users\<Windows User>\Documents\Projects` Windows folder, execute the following command: